example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -1103,8 +1115,10 @@ describe('The S3 Ingest Granules workflow', () => {
           'FilesToGranulesStep',
           'MoveGranuleStep',
           'UpdateGranulesCmrMetadataFileLinksStep',
+          'HyraxMetadataUpdatesTask',
```

Comment:
Because the test checks that each step in the workflow that's expected to run has input/output/etc.   It's validating all steps ran, however as it is an opt-in list, when new tasks are added, they have to be manually added here.

Given constraints on getting a demo out/etc I opted to scope limit this, but I may look into issuing a PR that uses local definitions or something that doesn't require a SF API call to generate this list instead of relying on this mechanism.